spec/controllers/participations_controller_spec.rb

```diff
@@ -69,7 +69,7 @@
       end
     end
 
-    context 'on a post you do not partecipate to' do
+    context 'on a post you do not participate to' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.